app/models/application_proceeding_type.rb

```diff
@@ -31,6 +31,28 @@ class ApplicationProceedingType < ApplicationRecord
     self.lead_proceeding = true if proceedings.empty?
   end
 
+  ##############################
+  # DELEGATED FUNCTIONS
+  # References to earliest delegated functions can be accessed off any application proceeding type
+  def used_delegated_functions?
+    earliest_delegated_functions.present?
+  end
+
+  def earliest_delegated_functions_date
+    proceedings.minimum('used_delegated_functions_on')
+  end
+
+  def earliest_delegated_functions_reported_date
+    earliest_delegated_functions.used_delegated_functions_reported_on
+  end
+
+  def earliest_delegated_functions
```

Comment:
Similar to above, `proceeding_with_earliest_delegated_functions` might be a clearer name.